tests/test_stream.py

```diff
@@ -24,6 +44,45 @@ def test_pretty_redirected_stream(httpbin):
     assert BINARY_SUPPRESSED_NOTICE.decode() in r
 
 
+def test_pretty_stream_ensure_full_stream_is_retrieved(httpbin):
+    env = MockEnvironment(
+        stdin=StdinBytesIO(),
+        stdin_isatty=False,
+        stdout_isatty=False,
+    )
+    r = http('--pretty=format', '--stream', 'GET',
+             httpbin.url + '/stream/3', env=env)
+    assert r.count('/stream/3') == 3
+
+
+@pytest.mark.parametrize('pretty', PRETTY_OPTIONS)
+@pytest.mark.parametrize('stream', [True, False])
+@responses.activate
+def test_pretty_options_with_and_without_stream_with_converter(pretty, stream):
+    plugin_manager.register(FakeConverterPlugin)
+    try:
+        assert 'FakeConverterPlugin' in str(plugin_manager)
```

Comment:
Why the string search instead of just `assert FakeConverterPlugin in plugin_manager`?

Reply:
It is to cover `PluginsManager.\_\_repr\_\_()`.